PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1522,6 +1522,11 @@ Do you want to continue?"/>
 			<UDLNewNameError title="UDL Error" message="This name is used by another language,
 please give another one."/>
 			<UDLRemoveCurrentLang title="Remove the current language" message="Are you sure?"/>
+			<UDL_importSuccessful title="User Defined Language" message="Import successful."/>
+			<UDL_importFails title="User Defined Language" message="Fail to import."/>
```

Comment:
Rather "Fail**ed** to import." or simply "Import fail**ed**." ?

Same in UserDefineDialog.cpp